Assets/Scripts/CombatManager.cs

```diff
@@ -66,22 +71,64 @@ public class CombatManager : MonoBehaviour {
     }
 
     void endCombat() {
-        Debug.Log("CombatManager:EndCombat()");
+        Debug.Log("CombatManager:endCombat()");
+        combatIsOver = true;
+
+        Pause(true); // stop combat from running and further game interaction
 
         combatGUI.Hide();
+
+        endCombatUI = Instantiate(endCombatUIPrefab);

```

Comment:
Does this ever get cleaned up?

Reply:
I believe when scene is unloaded (moving from level => menu) this gets Destroyed, but we could be more explicit